css/types/color.json

```diff
@@ -824,7 +829,8 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "17.5"
+                "version_added": "17.5",
+                "impl_url": "https://webkit.org/b/262914"
```

Comment:
Not sure if we generally want to have `impl_url` after the feature shipped but absent a guideline, this is fine.